Ordenamiento argumentativo

etalii opened the pull request.

etalii committed:
Update texto.md

etalii committed:
adecuación de los puntos 1 y 2

Adrianzo commented:
los links a issues y pull requests (PR) van en cuerpo de mensaje y no en título, por ejemplo este PR es en respuesta al issue #12 👍

Adrianzo committed:
corregí enlace partido

la sintaxis de markdown funciona en una misma linea, por ejemplo si pones negritas el par de dobles asteriscos debe estar en la misma linea, sino ya dejan de ser un par, son independientes uno del otro. Lo mismo pasa con los links, si pones algo entre corchetes y pegado la url entre los parentesis, no podes meter un enter xq rompes la sintaxis.

Adrianzo committed:
modifique contextos para que se estése mas pillo

Adrianzo committed:
agregue enlaces al indice y reworkie titulines

Adrianzo committed:
bueno, me voy a poner con el otro choclito q sigue
